microsite/pages/en/plugins.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+const fs = require('fs')
+const yaml = require('js-yaml');
+const React = require('react');
+const Components = require(`${process.cwd()}/core/Components.js`);
+const { Block: { Container }, BulletLine } = Components;
+
+const pluginsDirectory = require('path').join(process.cwd(), 'data/plugins');
+const pluginMetadata = fs.readdirSync(pluginsDirectory)
+  .map((file) => yaml.safeLoad(fs.readFileSync(`./data/plugins/${file}`, 'utf8')));
+const truncate = (text) => (text.length > 170 ? text.substr(0, 170) + '...' : text);
+
+const addPluginDocsLink = 'https://github.com/spotify/backstage/blob/master/microsite/README.md#adding-a-plugin-to-the-plugins-page';
+const defaultIconPath = 'img/logo-gradient-on-dark.svg';
+
+const Plugins = () => (
+    <main className="MainContent">
```

Comment:
Looks like you haven't run prettier here?

Reply:
It looks like the microsite doesn't have a linting or prettier config, which is why the pre-commit hook which lints and formats files didn't catch this.

We should certainly fix this as a once-off but linting and formatting should probably also be added to the pre-commit hook and the github action which checks pull requests to the microsite. @iain-b if I'm correct in my assessment, would you mind opening a separate issue for this please, if there isn't one already.

Reply:
Yep, I saw the todo list in the PR and went to run it and saw it wasn't there. I'm happy to run it on this file but I'd rather not touch files which I haven't modified functionally.